### PR TITLE
Adding option to manually modify the logo url 

### PR DESCRIPTION
**Pull request by spiderjako**

### CODE REVIEW

File: lib/sanbase/model/project/project.ex [hunk 1]
```diff
@@ -363,6 +365,12 @@ defmodule Sanbase.Model.Project do
     Repo.preload(projects, additional_preloads ++ @preloads)
   end
 
+  def upload_logo_url(%Project{} = project, logo_url) do
```
Comment (tspenov) [inline lib/sanbase/model/project/project.ex]: I think this function won't be needed if update is directly through ex_admin. Just make sure you add the dark logo field in edit form in ex_admin. When `update` is clicked it makes sure it goes through `changeset` function in project

File: priv/repo/structure.sql [hunk 2]
```diff
@@ -16,6 +16,20 @@ SET xmloption = content;
...
+CREATE EXTENSION IF NOT EXISTS plpgsql WITH SCHEMA pg_catalog;
+
+
+--
+-- Name: EXTENSION plpgsql; Type: COMMENT; Schema: -; Owner: -
+--
+
+COMMENT ON EXTENSION plpgsql IS 'PL/pgSQL procedural language';
+
+
```
Comment (IvanIvanoff) [inline priv/repo/structure.sql]: avoid committing psql extensions you have locally

Comment (spiderjako) [reply priv/repo/structure.sql]: Thanks, I thought I looked for such extensions, guess I've overlooked them

**Comment by tspenov** [comment]
LGTM. @spiderjako  @IvanIvanoff BTW how one will get the `s3` link of the logo - so he can update it through admin ?

File: test/sanbase_web/graphql/projects/project_api_test.exs [hunk 3]
```diff
@@ -118,6 +118,31 @@ defmodule SanbaseWeb.Graphql.ProjectApiTest do
...
   end
 
+  test "fetch project dark logos", context do
+    project =
+      insert(
+        :random_project,
+        %{
+          dark_logo_url: "dark_logo_url.png"
+        }
+      )
```
Comment (IvanIvanoff) [inline test/sanbase_web/graphql/projects/project_api_test.exs]: please fit this on 1 line

**Pull request merged**